modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberFocusTest.ts

```diff
@@ -189,4 +191,30 @@ describe('browser.tinymce.themes.silver.throbber.ThrobberFocusTest', () => {
     await pDisableThrobber(editor, pAssertEditorFocus(editor));
     assert.isTrue(editor.hasFocus());
   });
+
+  it('TINY-7602: should not steal focus if target element inside the editor body has "mce-pastebin" class', async () => {
+    const editor = hook.editor();
+    const editorBody = TinyDom.body(editor);
+    const normalDiv = SugarElement.fromHtml<HTMLElement>('<div class="not-pastebin" >&nbsp;</div>');
+    const fakePasteBin = SugarElement.fromHtml<HTMLElement>('<div class="mce-pastebin">&nbsp;</div>');
+
+    Insert.append(editorBody, normalDiv);
+    Insert.append(editorBody, fakePasteBin);
+    await pEnableThrobber(editor, pAssertThrobberFocus);
+    FocusTools.setFocus(SugarBody.body(), '#tempInput');
+
+    // Make sure throbber doesn't take focus when the pastebin is the target
+    editor.dom.fire(fakePasteBin.dom, 'focusin');
```

Comment:
Can we please not use fake events here and instead actually focus the element using `Focus.focus(fakePasteBin)`?

Reply:
I had tried that but was unable to get it to trigger the listeners set up in Throbber.ts

Reply:
I thought I had tried this before, but adding `tabindex` to the divs seems to allow `Focus.focus` to be used.

Reply:
The problem isn't really that from what I can see, it's that we're not setting up the pastebin correctly. If you look at https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/plugins/paste/main/ts/core/PasteBin.ts#L40 you'll see we add a number of other attributes (the contenteditable one specifically). To properly test this, we should be setting up a more realistic paste bin.

Reply:
In this case, it's likely the `contenteditable="true"` attribute that's needed to handle the focus call.

Reply:
Have added a note as to why we can't use `Focus.focus` here